style.css

```diff
@@ -22,13 +22,136 @@ body {
   margin-right: 24px;
 }
 
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  background-color: #fff;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+}
+
 .logo {
-  margin-right: auto;
-  margin-left: 24px;
+  font-size: 1.5rem;
+  font-weight: bold;
 }
 
-.desk {
-  display: none;
+.cover {
+  display: flex;
+  flex-direction: column;
+  width: 45%;
+  padding: 20px;
+  margin-left: -2%;
+  border-radius: 45%;
+  margin-bottom: 10%;
+}
+
+nav {
+  display: flex;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  display: inline;
+  left: 2.4%;
+  right: 88.27%;
+  top: 77.81%;
+  bottom: 18.88%;
+  margin-left: 15px;
+}
+
+nav li {
+  list-style: none;
+  display: inline;
+  left: 2.4%;
+  right: 88.27%;
+  top: 77.81%;
+  bottom: 18.88%;
+  margin-left: 15px;
+  margin-right: 1rem;
+}
+
+nav a {
+  text-decoration: none;
+  color: #333;
+  font-weight: bold;
+}
+
+.close-button .links {
+  height: 24px;
+  left: 4px;
+  right: 68px;
+  margin-top: 9%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.close-button {
+  position: absolute;
+  top: 0;
+  right: 8rem;
+  font-size: 2.8rem;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  margin-left: 81.9%;
+  color: #fff;
+}
+
+@media screen and (max-width: 768px) {
+  .menu-toggle {
+    display: block;
+  }
+
+  nav {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 97vh;
+    background-color: #6070fff2;
+    padding: 1rem;
+    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+    transform: translateX(-100%);
+    transition: transform 0.2s ease-in-out;
+  }
```

Comment:
- [ ] This is close as well but as you can see from the Figma design there's a blue effect on the background of the mobile menu so please make sure to implement it on your design as this will ensure a better consistency with the original design
- Figma design:
  ![image](https://github.com/mumbereh/my-portfolio/assets/104968236/4faf4a58-2187-4050-9fa1-653e31a8fa7d)
- Your design:
  ![image](https://github.com/mumbereh/my-portfolio/assets/104968236/4faf4a58-2187-4050-9fa1-653e31a8fa7d)